hack/test-cmd.sh

```diff
@@ -25,16 +25,57 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 source "${KUBE_ROOT}/hack/lib/test.sh"
 
+# Stops the running kubectl proxy, if there is one.
+function stop-proxy()
+{
+  [[ -n ${PROXY_PID-} ]] && kill ${PROXY_PID} 1>&2 2>/dev/null
```

Comment:
The general rule is to always use quotes.

In the `[[ ... ]]` expression it doesn't matter too much because `[[ ... ]]` is a bash built-in and word expansion does **not** happen inside it, but for the `kill` it would actually matter if it expands to more than one word.

As mentioned, quotes everywhere is usually the safest bet, except in places where you explicitly want to split the words (in which case, arrays are usually more appropriate.)

Cheers!
Filipe